.circleci/config.yml

```diff
@@ -9,6 +9,15 @@ jobs:
       - run: test/e2e-build.sh
       - run: test/e2e-tests.sh
 
+  e2e-supergloo-testing:
+    machine: true
+    steps:
+      - checkout
+      - run: test/e2e-kind.sh
+      - run: test/e2e-supergloo.sh
+      - run: test/e2e-build.sh supergloo:supergloo-system.test
```

Comment:
Can you please change the mesh provider format to  `supergloo:<MESH-NAME>.<NAMESPACE>` to follow the Kubernetes convention.